crates/figma_import/src/toolkit_schema.rs

```diff
@@ -105,6 +105,14 @@ impl Default for ScrollInfo {
     }
 }
 
+// This will be used to indicate which nodes are telltales, in the case
```

Comment:
Can you make this a doc comment? I think it's also worth generalizing the language a bit to say that the DesignCompose renderer implementation can use this hint to determine if it's important for the content to be rendered identically on different platforms.

Reply:
Updated comment